python/lib/communication/dmod/communication/maas_request/nwm/nwm_request.py

```diff
@@ -51,8 +51,9 @@ def __init__(
             super().__init__(**data)
         else:
             data["request_body"] = dict()
-            nwm_inner_request_body = {"config_data_id": config_data_id}
-            data["request_body"]["nwm"] = nwm_inner_request_body
+            data["request_body"]["nwm"] = data["model"]["nwm"]
```

Comment:
Either this or the unit tests are wrong - looks like `model` isn't coming in through `data`.

Reply:
@christophertubbs, nice catch, I just fixed it.